tests/Doctrine/Tests/ORM/Hydration/SingleScalarHydratorTest.php

```diff
@@ -76,10 +79,9 @@ public function testHydrateSingleScalar($name, $resultSet)
             $result = $hydrator->hydrateAll($stmt, $rsm);
             $this->assertEquals(1, $result);
         } else if ($name == 'result3' || $name == 'result4') {
```

Comment:
Can you change the code to use early returns? Also, this is an `in_array()` check :D